test/rekt/resources/broker/broker.go

```diff
@@ -113,3 +117,23 @@ func IsReady(name string, interval, timeout time.Duration) feature.StepFn {
 func IsAddressable(name string, interval, timeout time.Duration) feature.StepFn {
 	return k8s.IsAddressable(gvr(), name, interval, timeout)
 }
+
+// Address returns a broker's address.
+func Address(ctx context.Context, name string, interval, timeout time.Duration) (*apis.URL, error) {
+	var addr *apis.URL
+	err := wait.PollImmediate(interval, timeout, func() (bool, error) {
+		var err error
+		addr, err = k8s.Address(ctx, gvr(), name)
+		if err != nil || addr == nil {
+			if !apierrors.IsNotFound(err) {
```

Comment:
Shouldn't this be the other way around? When apierrors.IsNotFound(err) is true, err is nil. So either the if should be negated, or in the then block return false, err.

Reply:
and then fix line 133 as well

Reply:
Hm, yeah! fixed. thanks!